examples/closures/closures.bal

```diff
@@ -16,8 +16,8 @@ function basicClosure() returns (function (int) returns int) {
     return foo;
 }
 
-// Example function with multiple levels of lambda functions in which the
-// innermost lambda has access to all of its outer scope variables.
+// Example function with multiple levels of anonymous functions in which the
+// innermost anonymous function has access to all of its outer scope variables.
```

Comment:
Minor change:
"// Example function with multiple levels of anonymous functions where the
// innermost anonymous function has access to all of its outer scope variables."